tools/convert_partition.py

```diff
@@ -0,0 +1,232 @@
+import os
+import json
+import time
+import argparse
+import numpy as np
+import dgl
+import torch as th
+import pyarrow
+from pyarrow import csv
```

Comment:
Remember to add package dependencies in the README.

Reply:
in which README?